### DISCUSSION TITLE
How do I update Grafana ? (error message)

### DISCUSSION THREAD

**Discussion by martin86x** [tteck/Proxmox#2752] [Help Needed]
hello everyone how do I upgrade grafana in Proxmox?

when I open the shell

die install code 
 bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/ct/grafana.sh)" 

and confirm the update I get the following message:
  ______           ____                 
  / ____/________ _/ __/___ _____  ____ _
 / / __/ ___/ __  / /_/ __  / __ \/ __  /
/ /_/ / /  / /_/ / __/ /_/ / / / / /_/ / 
\____/_/   \__,_/_/  \__,_/_/ /_/\__,_/  
 
 - Updating Grafana  |
[ERROR] in line 59: exit code 0: while executing command apt-get update &> /dev/null

My version is the [v9.2.3 (ec9c2ed029)

**Comment by tteck** [comment]
🌐 It seems that you're facing a network problem, _probably_ linked to DNS.

**Reply by martin86x** [reply]
My DNS server should not be the problem it automatically assigns other scripts work great I just installed CasaOS with it

**Comment by tteck** [comment]
English only please.

**Reply by martin86x** [reply]
ok i change it
